src/Cameras/targetCamera.ts

```diff
@@ -312,11 +312,19 @@ export class TargetCamera extends Camera {
 
         // Rotate
         if (needToRotate) {
+            //rotate, if quaternion is set and rotation was used
+            if (this.rotationQuaternion) {
+                this.rotationQuaternion.toEulerAnglesToRef(this.rotation);
+            }
+
             this.rotation.x += this.cameraRotation.x * directionMultiplier;
             this.rotation.y += this.cameraRotation.y * directionMultiplier;
 
+            console.log(this.cameraRotation);

```

Comment:
You probably want to remove the `console.log` (2 occurrences)

Reply:
Mouahahahahah... Yup pushing in a sec, thanks a ton.